test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/testproviders/PersistenceTestProvider.java

```diff
@@ -74,6 +73,7 @@ public void testControllerPersistence(WorkbenchKieServerScenario deploymentScena
         String kieServerLocation = serverInfo.getLocation();
         try {
             WorkbenchUtils.saveContainerSpec(kieControllerClient, serverInfo.getServerId(), serverInfo.getName(), containerId, containerId, Kjar.DEFINITION, KieContainerStatus.STARTED);
+            KieServerClientProvider.waitForContainerStart(deploymentScenario.getKieServerDeployment(), containerId);
```

Comment:
I added this because sometimes this test failed. It seems we're ensuring to not move forward until the container starts in the kie server.